CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix `Code.UnmarshalJSON` to work with valid json only. (#1276)
 - The `resource.New()` method changes signature to support builtin attributes and functional options, including `telemetry.sdk.*` and
   `host.name` semantic conventions; the former method is renamed `resource.NewWithAttributes`. (#1235)
+- Fix condition in `label.Any`. (#1299)
```

Comment:
We probably don't need a Changelog entry for this given this fixes something that has yet to be released.

If we do want to include one can we move this entry to the `Fixed` section?